lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -148,6 +148,33 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
   final Map<Object, ui.ViewConfiguration> _windowConfigurations =
       <Object, ui.ViewConfiguration>{};
 
+  /// The implicit [FlutterView] provided by the platform, if any.
+  ///
+  /// If the platform provides an implicit view, it can be used to bootstrap
+  /// the framework. This is common for platforms designed for single-view
+  /// applications (e.g. on mobile devices with a single display).
+  ///
+  /// Applications and libraries must not rely on this property being set
+  /// as it may be null depending on the engine's configuration. Instead,
+  /// consider using [View.of] to lookup the [FlutterView] the current
+  /// [BuildContext] is drawing into.
+  ///
+  /// While the properties on the referenced [FlutterView] may change,
+  /// the reference itself is guaranteed to never change over the lifetime
+  /// of the application: If this property is null at startup, it will remain
+  /// so throughout the entire lifetime of the application. If it points to a
+  /// specific [FlutterView], it will continue to point to the same view until
+  /// the application is shut down (although the engine may replace or remove
+  /// the underlying backing surface of the view at its discretion).
+  ///
+  /// See also:
+  ///
+  /// * [View.of], for accessing the current view.
+  /// * [PlatformDisptacher.views] for a list of all [FlutterView]s provided
+  ///   by the platform.
```

Comment:
Just a note to remember to update here to match the lib/ui version.

Reply:
Actually, I think you can just delete this copy of the doc? It should just get inherited from the super method.

Reply:
@ditman Do you know why web's engine `PlatformDispatcher` has the doc comments? Is it safe to remove this comment?

Reply:
> Just in case it's not clear, this overrides the web_ui's PlatformDispatcher, not ui's `PlatformDispatcher.

I did indeed miss that.

Still, I don't think the doc comments from web_ui are used anywhere (well, except maybe for people reading the source directly). I defer to the web team to decide whether these comments should actually be duplicated here.

Reply:
> Do you know why web's engine PlatformDispatcher has the doc comments? Is it safe to remove this comment?

Not really, I remember some pushes in the past to remove documentation from web_ui that had drifted with regards to the proper dart:ui version (which is the one that is published in the docsite, anyways), and letting the IDE pick it up as suggested by @goderbauer.

(Unfortunately I cannot find a specific commit doing what I describe, so maybe it was done in small increments (sorry!))

If the IDE picks up the documentation from dart:ui, I'm fine with it being removed from the web_ui source.